### DISCUSSION TITLE
How to use exist scoop installation?

### DISCUSSION THREAD

**Discussion by F-TD5X** [ScoopInstaller/Scoop#5106] [Help and Questions] [answered]
Such as when I restore my windows, or just pack the installation to ship to my frinds.

What ever but we lost all the env and registry.

And how can I import thre exist installation?

I have try to set new env of SCOOP_GLOBAL and SCOOP, install scoop and then run `scoop reset *`.

This can fix most of problems, but, I have to restore some config file after this steps. The original config file will be renamed for conflict.

Are there some better way to use the exist installation?

**Comment by lewis-yeung** [comment, marked as answer]
Learn and try `scoop export`/`scoop import` to see if they help. Some app configs like the registry should be backed up and restored manually.
